e2e/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "no-color-literals": 0,
     "no-console": 0,
     "@typescript-eslint/no-namespace": "off",
-    "cypress/no-async-tests": "error"
+    "cypress/no-async-tests": "error",
+    "quotes": [2, "double", { "avoidEscape": true }]
```

Comment:
1. I personally don't like using numbers since I am always confused about which number refers to which configuration. Do you think using string `"error"` would be more concrete?
2. Why don't we do this for the whole project too? i.e. put this setting in `.eslintrc` in the root directory instead. Why do we want to do this specifically only for `e2e` directory?